students/BrianE/session03/list_lab.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/env python3
+
+"""
+    list_lab.py
+"""
+
+
+def display_fruits(fruits):
+    """
+    Print fruit list contents to terminal
+    :param fruits: list of fruits
+    :return: None
+    """
+
+    print("Current fruit: {}".format(fruits))
+
+
+def delete_fruits(fruits):
+    """
+    Delete fruit item from list
+    :param fruits: string
+    :return: None
+    """
+
+    print(fruits)
+    fruit_to_delete = input("Enter a fruit to delete: \n\n >>>")
+    try:
+        index = fruits.index(fruit_to_delete)
+        del fruits[index]
+    except ValueError as e:
+        print(e)
+
+
+def cleanup_list(fruits):
+    """
+    Remove fruits from list if user does not like
+    :param fruits: list of fruits
+    :return: None
+    """
+
+    deleted_fruits = []
+
+    for i in range(0, len(fruits)):
+        while True:
+            answer = input(f"Do you like {fruits[i]}?").lower().strip()
+            if answer == 'yes':
+                break
+            elif answer == 'no':
+                deleted_fruits.append(fruits[i])
+                break
+            else:
+                print(f"Please choose from the following: yes|no")
+                continue
+
+    for fruit in deleted_fruits:
```

Comment:
This does work! You will just want to keep in mind that when working with large datasets, creating a secondary list could consume a lot of memory.